src/shared/components/DefaultLayout/index.tsx

```diff
@@ -8,13 +8,18 @@ interface Props {
 }
 
 const DefaultLayout = ({ main, header, footer }: Props) => (
-  <div className="flex flex-col justify-between h-screen">
-    {header && <header>{header}</header>}
-    <main id={ELEMENT_ID.main} className="w-full flex-1 relative overflow-scroll scrollbar-hide">
+  <div className="flex flex-col min-h-screen w-full">
+    {header && (
+      <header className="sticky top-0 max-w-[600px] w-full mx-auto z-header">{header}</header>
+    )}
+    <main id={ELEMENT_ID.main} className="max-w-[600px] w-full mx-auto relative flex-1">
```

Comment:
```suggestion
  <div className="w-full max-w-[600px] mx-auto shadow-lg">
    {header && (
      <header className="w-full sticky top-0 z-header">{header}</header>
    )}
    <main id={ELEMENT_ID.main} className="w-full relative flex-1">
```
여기는 이렇게 하는 건 어떨까요??
shadow도 있었으면 해요.

Reply:
아하 shadow 적용 했습니다!